src/Microsoft.AspNetCore.Http.Connections/HttpConnectionOptions.cs

```diff
@@ -3,14 +3,15 @@
 
 using System.Collections.Generic;
 using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http.Connections.Internal;
 
 namespace Microsoft.AspNetCore.Http.Connections
 {
     public class HttpConnectionOptions
     {
         public IList<IAuthorizeData> AuthorizationData { get; } = new List<IAuthorizeData>();
 
-        public HttpTransportType Transports { get; set; } = HttpTransportType.All;
+        public HttpTransportType Transports { get; set; } = HttpTransports.All;
```

Comment:
Pubternal types in public namespace? Naw.

Reply:
This syntax is compiled down so that the value is set inside the `HttpConnectionOptions` ctor I thought?

Reply:
It is but I updated class to use ctor to set defaults to make it more obvious.